src/index.html

```diff
@@ -0,0 +1,69 @@
+<html>
+  <head>
+    <link rel="stylesheet" href="css/vote-CSS.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Varela+Round&display=swap'"
```

Comment:
Is there a rouge `'` just before the ending quote?

Reply:
Apologise! Mistake on my side